README.md

```diff
@@ -32,42 +31,132 @@ Everything is co-ordinated through github issues.  The current development proce
  Check out the full list [issues page](https://github.com/Western-Health-Covid19-Collaboration/wh_covid19_app/issues) to get an overview of everything that is happening.  Feel free to flag an issue with `design` at any time to summon a designer.
  
 ⬇️
- 
-🖥 **CI** builds the app and deploys to phones via firebase app distribution.
 
+🖥 **CI** builds the app and deploys to phones via firebase app distribution.
 
 ## The Plan
 
 The app will have these screens and sections:
 
 ![App information architecture](docs/Flow-Diagram.png "The information architecture of the app")
 
-We will rapidly build out functionality and screens.  The plan is to ship early, and ship often, deploying new builds at least once a day via CI.
+We will rapidly build out functionality and screens. The plan is to ship early, and ship often, deploying new builds at least once a day via CI.
+
+Currently all data will be stored inside the app, with no CMS or web services. The app should have minimal logic, and instead just focuses on providing important information in a mobile friendly format.
+
+## Getting Started & Development
+
+You will need to install and setup Flutter to develop this application. You can read more on how to set up Flutter [here](https://flutter.dev/docs/get-started/install).
+
+**Tip!**: If you are using Visual Studio Code as your IDE (Code Editor), the [Flutter extension](https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter) is really useful and helpful to perform multiple actions such as:
+
+- the installation of depencies
+- running the application
+
+### Installing dependencies
+
+To install dependencies, simply run `flutter pub get` in your terminal.
+
+### Developing the app
+
+#### iOS
+
+Initialise an instance of an iOS Simulator or connect physical iOS device, navigate to the cloned repo and run from root:
+
+```sh
+# from terminal
+
+open -a simulator
+
+cd desktop/muhnee-app
+flutter run -d "iPhone"
+```
+
+#### Android
+
+##### Keychain Certificate Setup
+
+1. Download the `google-services.json` configuration file from the Firebase Project and move it to the directory `<PROJECT-ROOT>/android/app`
+2. Get the Android Keychain by:
+
+   ```
+   keytool -list -v -keystore ~/.android/debug.keystore -alias androiddebugkey -storepass android -keypass a
+   ndroid
+   ```
+
+3. Copy either the SHA1 or the SHA256 certificate fingerprint onto the Android App on [Firebase Console]()
+
+##### Development
+
+If you have a device plugged in just run:
+
+```
+flutter run
+```
+
+Otherwise if you have setup [Android Studio](https://developer.android.com/studio) and/or the Android Software Development Kit (SDK) you can run it over a virtual Android emulator using the Android Debug Bridge (ADB). Just simply run:
+
+```
+flutter run adb
+```
+
+##### Releasing
```

Comment:
Please remove docs about releasing, that process  is only and can only be done on CI

Reply:
Done!